Makefile

```diff
@@ -51,15 +53,13 @@ lint: black flake8 mypy sdist-check
 
 # ---- Unit Tests (no algod) ---- #
 
-# TODO: add blackbox_test.py to multithreaded tests when following issue has been fixed https://github.com/algorand/pyteal/issues/199
-
-NUM_PROCS = auto
 test-unit-async:
-	pytest -n $(NUM_PROCS) --durations=10 -sv pyteal tests/unit -m "not serial"
+	pytest -n auto --durations=10 pyteal tests/unit -m "not serial"
 
+# Run tests w/ @pytest.mark.serial under ~/tests/unit each in its own proc:
 test-unit-sync:
-	pytest --durations=10 -sv pyteal tests/unit -m serial
-
+	find tests/unit -name '*_test.py' | sort | xargs -t -I {} pytest --suppress-no-test-exit-code --dist=no --durations=10 {} -m serial
+ 
```

Comment:
why `test-*-sync` are so different from `test-*-async`? I thought you just need to specify the `serial`...